go-controller/pkg/libovsdbops/model_client.go

```diff
@@ -306,6 +308,20 @@ func (m *ModelClient) create(opModel *OperationModel) ([]ovsdb.Operation, error)
 			Until:   "!=",
 			Rows:    []ovsdb.Row{{"name": opModel.Name}},
 		})
+	} else if o[0].Table == "Logical_Router_Policy" {
+		lrp, ok := opModel.Model.(*nbdb.LogicalRouterPolicy)
+		if !ok {
+			return nil, fmt.Errorf("expected LogicalRouterPolicy model, got: %v", opModel.Model)
+		}
+		ops = append(ops, ovsdb.Operation{
+			Op:      ovsdb.OperationWait,
+			Timeout: &timeout,
+			Table:   o[0].Table,
+			Where:   []ovsdb.Condition{{Column: "match", Function: ovsdb.ConditionIncludes, Value: lrp.Match}},
```

Comment:
Let's make sure in all the LRP cases, the match is always unique.. (from all the policies we have so far, it seems to be the case, but just want to make sure we won't have a case where same match can have more than 1 nexthop or some weird case)

Reply:
Good point. I think in the case "nexthops" is part of the action and we wouldn't want to create things with multiple match criteria that matches different nexthops. So excluding them from this check is OK. However, there could be cases where  the same match criteria is created for other features with different priorities. In that case we would want both to get configured, so I think we need to include the priority here as well.

Reply:
Added